plugin.video.reddit_viewer/resources/language/English/strings.xml

```diff
@@ -0,0 +1,135 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<strings>
+  <string id="30001">Add subreddit</string>
```

Comment:
In your next update, please convert strings.xml to strings.po files. strings.xml is deprecated.